Apromore-Core-Components/Apromore-Manager/src/main/resources/db/migration/changeLog.yaml

```diff
@@ -0,0 +1,27 @@
+databaseChangeLog:
+ - include:
+    file: db/migration/changeLog-Schema-init.yaml
+
+ - changeSet:  
+    id: 1599618934768-128  
```

Comment:
Initial id: too long. OK if it is auto-increment. If manually, can we make it 1, 2, 3...?

Reply:
Well thhat ID is not to be managed, its nothing to do with sequence, also those ids for the initial script is autogenerated by liquibase.
I will be adding user meta data next to the changelog, It will be a seperate file where we will start adding schema and data (if required). The above file is not meant to be touched in the future.
the Ids to be used by devs will be YYYYMMDDHHmmSS,  This will tell us when the query is added. However there is no hard and fast rule.

Reply:
OK, understanding these two initial changeSets are one-off and other changeSets in the future will use a different ID format.

Reply:
@nolantellis Perhaps you could outline a protocol/general rule to add changeSet in the confluence page. E.g.:
- Change set is immutable/never edit
- Create a new file for a new change, name it YYYYMMDDHHmmSS (Australian Eastern Standard Time), changeSet id is the same (or plus sequence id for multiple sets, id: 1599618934768-1)
- Must be merged via PR (note that we don't have code review in place right now)

Reply:
It might be a good idea if the two initial changeSets use the above format (YYYYMMDDHHmmSS) as setting the first example.

Reply:
Yesy, I will be doing this for user meta data which was added earlier but not yet executed in production. That will clear a lot of doubts